install/ansible/istio/tasks/install_addons.yml

```diff
@@ -1,41 +1,21 @@
-- include_tasks: add_serviceaccount_to_addon.yml
-    add_on_name="{{item}}"
-  with_items: "{{ selected_addons_needing_sa }}"
-  when: cluster_flavour == 'ocp'
+- name: Install Kiali
+  block:
+  
+  - set_fact: 
+      kiali_version: 0.3.1.Alpha
 
-- include_tasks: install_istio_addons.yml
-  when: is_istioaddon_iterable
+  - name: Install Kiali on Openshift
+    shell: |
+      curl https://raw.githubusercontent.com/kiali/kiali/{{ kiali_version }}/deploy/openshift/kiali-configmap.yaml | VERSION_LABEL={{ kiali_version }} envsubst | {{ cmd_path }} create -n {{ istio_namespace }} -f -
+      curl https://raw.githubusercontent.com/kiali/kiali/{{ kiali_version }}/deploy/openshift/kiali.yaml | IMAGE_NAME=kiali/kiali IMAGE_VERSION={{ kiali_version }} NAMESPACE={{ istio_namespace }} VERSION_LABEL=master VERBOSE_MODE=4 envsubst | {{ cmd_path }} create -n {{ istio_namespace }} -f -
```

Comment:
Kiali will soon have a third file (for secrets) - see https://github.com/kiali/kiali/pull/305

Reply:
Shouldn't be a problem. We'll either need to update this PR if the other lands after the Kiali release, or issue another one if it's the other way around

Reply:
@jmazzitelli the current playbook deploys from the `0.3.1.Alpha` branch. Which one should actually be used? @gbaufake WDYT?

Reply:
We should be tagging a new Kiali release sometime soon. Once we do that, I would suggest we wait and update this PR to pull that new version when it gets pushed (will probably be something like "v0.4.0").

Reply:
OK, there is a v0.4.0 docker image that should be used. See https://hub.docker.com/r/kiali/kiali/tags/

Reply:
@jmazzitelli PR updated to use the latest Kiali version